src/scales/scale.radialLinear.js

```diff
@@ -512,26 +543,35 @@ class RadialLinearScale extends LinearScaleBase {
 		}
 
 		const startAngle = me.getIndexAngle(0);
-		const tickFont = toFont(tickOpts.font);
 		let offset, width;
 
 		ctx.save();
-		ctx.font = tickFont.string;
 		ctx.translate(me.xCenter, me.yCenter);
 		ctx.rotate(startAngle);
 		ctx.textAlign = 'center';
 		ctx.textBaseline = 'middle';
 
 		me.ticks.forEach((tick, index) => {
+			const context = {
+				chart: me.chart,
+				scale: me,
+				index,
+				tick,
+			};
+			const tickFont = me._resolveTickFontOptions(index);
+			ctx.font = tickFont.string;
```

Comment:
should probably set (and resolve) the font after the possible `return` below.

Reply:
Good call. I've made that update

Reply:
I think you lost the last commit in rebase